www/about/policies/terms-of-service.md

```diff
@@ -38,60 +38,59 @@ these Terms.
 1. The Gratipay Model and General Rules
 
   1. The Service is a platform to enable Teams of Gratipay Participants to
-     receive payments to fund Open Work. Open Work means that the Team provides
+     receive payments to fund **Open Work. Open Work means that the Team provides
```

Comment:
This definition of "Open Work" needs to be modified to include transparent work, or removed and "work" left in its place.